### ISSUE TITLE
ValueError: unsupported pickle protocol: 5

### ISSUE THREAD

**Issue by NikolayKolibarov** [instabot-py/instabot.py#2595]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [x] If so, have you tried running the code on your local machine?

```
Include more details about the environment
```

#### Operating System? (include version)
- [x] macOS
- [ ] Windows
- [ ] Rasbian Linux Raspberry PI (include flavour)
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

```
include the output of 'uname -a'
Darwin Nikolays-MacBook-Pro.local 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_64
```

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
Include here the response of 'python --version' or 'python3 --version'
```

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
pip 20.0.2 from /Users/nikolaykolibarov/Dev/anaconda3/lib/python3.7/site-packages/pip (python 3.7)
```

#### Instabot Version?
```
Include here the response of 'instabot-py --version'
Installed version 0.7.10
```

#### How you use the instabot
- [x] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue
I have used the bot before. I always have problems to tackle. When I try to run the bot, it starts and then fails. Here is the output:
```
Traceback (most recent call last):
  File "/Users/nikolaykolibarov/Dev/anaconda3/bin/instabot-py", line 10, in <module>
    sys.exit(main())
  File "/Users/nikolaykolibarov/Dev/anaconda3/lib/python3.7/site-packages/instabot_py/__main__.py", line 357, in main
    bot = InstaBot(config=config)
  File "/Users/nikolaykolibarov/Dev/anaconda3/lib/python3.7/site-packages/instabot_py/instabot.py", line 219, in __init__
    self.login()
  File "/Users/nikolaykolibarov/Dev/anaconda3/lib/python3.7/site-packages/instabot_py/instabot.py", line 297, in login
    cookies = pickle.load(i)
ValueError: unsupported pickle protocol: 5
```
I hope someone can help out. Thanks in advance!

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

#### Config

```
Ensure that you specify the configuration file in the command line.
Ensure that the default file instabot.config.yml contains only common settings if you use multiple accounts.
Include the contents of your config here (instabot-py -c config.yml --dump) (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT

```

**Comment by TheNazareen** [comment]
i have the same problem

**Comment by nickgryg** [comment]
The issue has been fixed in version `0.7.12`.
